monkey/infection_monkey/exploit/tools/helpers.py

```diff
@@ -43,6 +43,12 @@ def get_target_monkey_by_os(is_windows, is_32bit):
 
 def build_monkey_commandline_explicitly(parent=None, tunnel=None, server=None, depth=None, location=None):
     cmdline = ""
+    
+    from infection_monkey.config import WormConfiguration                              #Passing Multiple Servers for Monkey to connect back
+    Server_String =" "
```

Comment:
why does it start from upper case letters? Also, no space after `=`. You should run linting software or use formatting capabilities of your IDE. Most can format code automatically and make sure things like this doesn't happen.